doc/using/Foreign.rst

```diff
@@ -62,6 +62,25 @@ The value of the attribute must start with ``VHPIDIRECT`` (an
 upper-case keyword followed by one or more blanks). The linkage name of the
 subprogram follows.
 
+The object file with the source code for the foreign subprogram must then be 
```

Comment:
I don't think it's possible to rellocate this content here. The hierarchy of the content is:

- Foreign declarations
  - Restrictions on foreign declarations
- Linking foreign object files to GHDL
- Wrapping and starting a GHDL simulation from a foreign program
- ...

You are changing it to:

- Foreign declarations
- Linking foreign object files to GHDL
  - Restrictions on foreign declarations
- Wrapping and starting a GHDL simulation from a foreign program
- ...

"Linking..." needs to be either before "Foreign declarations" or after "Restrictions on foreign declaration".